internal/commands/servergroup/show.go

```diff
@@ -0,0 +1,126 @@
+package servergroup
+
+import (
+	"strconv"
+
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/commands"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/completion"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/format"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/labels"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/output"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/resolver"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/ui"
+
+	"github.com/UpCloudLtd/upcloud-go-api/v6/upcloud"
+	"github.com/UpCloudLtd/upcloud-go-api/v6/upcloud/request"
+)
+
+// ShowCommand creates the "servergroup show" command
+func ShowCommand() commands.Command {
+	return &showCommand{
+		BaseCommand: commands.New(
+			"show",
+			"Show server group details",
+			"upctl servergroup show 8abc8009-4325-4b23-4321-b1232cd81231",
+			"upctl servergroup show my-server-group",
+		),
+	}
+}
+
+type showCommand struct {
+	*commands.BaseCommand
+	resolver.CachingServerGroup
+	completion.ServerGroup
+}
+
+// Execute implements commands.MultipleArgumentCommand
+func (c *showCommand) Execute(exec commands.Executor, uuid string) (output.Output, error) {
+	svc := exec.All()
+	serverGroup, err := svc.GetServerGroup(exec.Context(), &request.GetServerGroupRequest{UUID: uuid})
+	if err != nil {
+		return nil, err
+	}
+
+	groupStatus := notApplicable
+
+	statusMap := make(map[string]string, 0)
+	if serverGroup.AntiAffinityPolicy != upcloud.ServerGroupAntiAffinityPolicyOff {
+		for _, serverState := range serverGroup.AntiAffinityStatus {
+			status := string(serverState.Status)
+			statusMap[serverState.ServerUUID] = status
+
+			if groupStatus != unMet {
+				groupStatus = status
+			}
+		}
+	}
+
+	serverRows := []output.TableRow{}
+	for _, serverUUID := range serverGroup.Members {
+		serverDetails, err := svc.GetServerDetails(exec.Context(), &request.GetServerDetailsRequest{UUID: serverUUID})
```

Comment:
I think in some commands (at least `network show`) we leave out details from the related row, if getting details for some resource fails. Not sure if it's possible for subaccount to see that server is part of server group, but not have access to that server.

Reply:
Tested this now and seems that subaccount only has access to servergroup if it has permissions to all of its servers. So this should not cause any issues 👌

Reply:
Excellent, thanks for the comments and testing!